backend/engine/utils/plugin.py

```diff
@@ -8,27 +8,26 @@
 from urllib.parse import quote_plus
 
 import boto3
-from botocore.exceptions import ClientError
-from django.db.models import Q
-
 from artemisdb.artemisdb.models import PluginConfig
 from artemislib.github.app import GITHUB_APP_ID
-from artemislib.logging import Logger, LOG_LEVEL
+from artemislib.logging import LOG_LEVEL, Logger
 from artemislib.util import dict_eq
+from botocore.exceptions import ClientError
+from django.db.models import Q
```

Comment:
These should not have been moved. `isort` issue?

Reply:
Good catch. I think I've fixed what was causing this, too.